src/Rectangle.java

```diff
@@ -0,0 +1,21 @@
+
+public class Rectangle {
+    public Rectangle(int lengthOfObject, int widthOfObject) {
+        lengthOfObject = 0;
+        widthOfObject = 0;
+    }
+
+    public int Area (int lengthOfObject, int widthOfObject) {
+    int area = 0;
+    area = lengthOfObject * widthOfObject;
+    return area;
+    }
```

Comment:
можна зразу результат передавати, всі методи ми називаємо з маленької букви. підправ це всюди
```suggestion
    public int area (int lengthOfObject, int widthOfObject) {
       return lengthOfObject * widthOfObject;
    }
```